examples/tree_1d_dgsem/elixir_euler_convergence_implicit_sparse_jacobian.jl

```diff
@@ -0,0 +1,151 @@
+using Trixi
+using OrdinaryDiffEqSDIRK
+
+# Functionality for automatic sparsity detection
+using SparseDiffTools, Symbolics
+
+# We need to avoid if-clauses to be able to use `Num` type from Symbolics.
+# In the Trixi implementation, we overload the sqrt function to first check if the argument 
+# is < 0 and then return NaN instead of an error.
+# To turn off this behaviour, we switch back to the Base implementation here
+# Redirect the warning to stdout to satisfy the CI tests which fail for anything in stderr
+redirect_stderr(stdout) do  
+    Trixi.set_sqrt_type!("sqrt_Base")
+end
```

Comment:
Just want to call out explicitly that I had to redirect stderr to stdout here because the CI tests error on anything in stderr (except a few explicitly ignored lines).

I originally wanted to add the warning string that's printed here to the `additional_ignore_content` kwarg of `@test_trixi_include` however `test_trixi_include` passes all the kwargs to `trixi_include` which will try to override any variables passed in the kwargs. 

So the alternative to redirectign stderr here would be to add the `additional_ignore_content` kwarg to `@test_trixi_include` and then to just create a dummy variable in this elixir called `additional_ignore_content` so that `trixi_include` is happy and so that the warning is ignored in the test. Let me know if that sounds better